packages/lib/server/queries/teams/index.ts

```diff
@@ -422,8 +423,56 @@ export async function isTeamMember(userId: number, teamId: number) {
   }));
 }
 
-export async function updateNewTeamMemberEventTypes(userId: number, teamId: number) {
-  const eventTypesToAdd = await prisma.eventType.findMany({
+export function generateNewChildEventTypeDataForDB({
```

Comment:
Refactor out how we generate the data used to create children managed event types.